utils/bundles.stanza

```diff
@@ -140,6 +143,7 @@ public pcb-bundle spi  (pins:Tuple<SPIPins>):
       SPI-CS  : make-pin(`cs)
       SPI-CIPO  : make-pin(`cipo)
       SPI-COPI  : make-pin(`copi)
+      SPI-SDIO  : make-pin(`sdio)
```

Comment:
Where does this pin come from? Is it in standard SPI?

Reply:
It looks like a standard SPI. It is present in many files and it shows up as 
```
STM32WLE5CBUx.xml:		<Signal Name="SPI1_NSS"/>
STM32WLE5CBUx.xml:		<Signal Name="SPI2_NSS"/>
STM32MP151CAAx.xml:		<Signal Name="SPI5_NSS"/>
```

Reply:
I was asking about `SDIO` . NSS for SPI you already capture and map to spi.cs, correct?

Reply:
My description in PR is incorrect. I added a new NSS pin for UART and USART. NSS in SPI maps to cs and that has not changed

Reply:
OK. NSS should show up in JITX as CS, so UART-CS and USART-CS should be the pins in bundles.stanza.

Still curious about SDIO in SPI as a separate issue.

Reply:
First about NSS. I will rename the pin. The current pin names in UART/USART are
```
  UART-DTR
  UART-CTS
  UART-DCD
  UART-RI
  UART-DST
  UART-RTS
  UART-CK
  UART-DE
  UART-NSS
```

I will change UART-NSS to UART-CS. Any other changes? Do any of the other pins also map to UART-CS?

Second about SDIO. I have not seen it appear anywhere in STM files. The only things show up are sck, sdo, sdi and sdo. Please let me know if we should change anything else